runelite-mixins/src/main/java/net/runelite/mixins/RSMainBufferProviderMixin.java

```diff
@@ -42,6 +42,6 @@
 	@Replace("draw")
 	final void draw(Graphics graphics, int x, int y)
 	{
-		Hooks.draw(this, graphics, x, y);
+		Hooks.draw(this, graphics);
```

Comment:
What is the point of this commit?

Reply:
Consistency with other draw-related methods, it does not need x/y so only needed fields should be exposed